styles.css

```diff
@@ -1,47 +1,60 @@
 * {
   box-sizing: border-box;
+  font-family: poppins, sans-serif;
+  margin: 0;
+  padding: 0;
 }
 
-body {
-  display: flex;
-  font-family: poppins, sans-serif;
-  flex-direction: column;
-  margin: 0 auto;
-  width: 375px;
+main {
+  max-width: 400px;
```

Comment:
- [ ] [OPTIONAL] I **HIGHLY** recommend you to not fix the width of the main. It will prevent your webpage to be responsive 📱 💻